config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -130,6 +136,187 @@ presubmits:
         env:
         - name: CSI_PROW_KUBERNETES_VERSION
           value: "latest"
+        - name: CSI_PROW_TESTS
+          value: "sanity serial parallel"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
+          # during the tests more like 3-20m is used
+          cpu: 2000m
+  - name: pull-kubernetes-csi-csi-driver-host-path-alpha-1-13-on-kubernetes-1-13
+    # Experimental job, explicitly needs to be started with /test.
+    # This can be enabled once the components have the necessary configuration
+    # for this combination of deployment+Kubernetes.
+    always_run: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - ./.prow.sh
+        env:
+        # We pick some version for which there are pre-built images for kind.
+        # Update only when the newer version is known to not cause issues,
+        # otherwise presubmit jobs may start to fail for reasons that are
+        # unrelated to the PR. Testing against the latest Kubernetes is covered
+        # by periodic jobs (see https://k8s-testgrid.appspot.com/sig-storage-csi#Summary).
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "1.13.3"
+        - name: CSI_PROW_KUBERNETES_DEPLOYMENT
+          value: "1.13"
+        - name: CSI_PROW_TESTS
+          value: "serial-alpha parallel-alpha"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2000m
+  - name: pull-kubernetes-csi-csi-driver-host-path-alpha-1-13-on-kubernetes-master
```

Comment:
Alpha suites should not run cross versions

Reply:
I guess for an alpha pull job, we should only have master->master

Reply:
> Alpha suites should not run cross versions.

They can for known configurations (like 1.13 on 1.14), because for those we can configure which tests are expected to work. But if we don't want that, we can also leave those alpha tests optional.

For all `-on-kubernetes-master` jobs the intention was to never make these tests mandatory pre-submit checks, because there's no guarantee that this check doesn't get broken by changes in master, see https://github.com/kubernetes/test-infra/blob/04c6ebf9292a2029f73997c38d7c9ceb894240ea/config/jobs/kubernetes-csi/gen-jobs.sh#L204-L206

Reply:
I don't think we should encourage providing more alpha support than our policy

Reply:
I doubt that many people besides the core team will know that these jobs are possible, so I'm not sure who or what that would encourage. But I don't have a strong opinion about those, so I can also take them out.

Reply:
Done. Also fixed some settings for the canary jobs.